index.html

```diff
@@ -1,9 +1,63 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Saira:ital,wght@0,100..900;1,100..900&display=swap"
+      rel="stylesheet"
+    />
+    <link href="css/headerFooter.css" rel="stylesheet" />
+    <link href="css/aboutUs.css" rel="stylesheet" />
+
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Group Project Title</title>
   </head>
-  <body></body>
+  <body>
+    <header class="header">
+      <div id="paw"></div>
+      <a href="#" id="aboutUs">About us</a>
+      <a href="#" id="menu">Menu</a>
+      <a href="#" id="careers">Careers</a>
+      <a href="#" id="buttonOrder">ORDER NOW</a>
+    </header>
+    <main>
+      <div class="container">
+        <div class="about-founder">
+          <img src="../assets/svg/original.svg" alt="Bear" class="bear-image" />
```

Comment:
Wrong reference for images, it should start  with "./"

![Capture](https://github.com/user-attachments/assets/aed2ed05-2071-4c04-872b-6ac469ad8c2d)

Reply:
fixed